Index: src/till/Table.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/till/Table.java b/src/till/Table.java
--- a/src/till/Table.java	
+++ b/src/till/Table.java	
@@ -11,6 +11,7 @@
     private int seats;
     private Reservation booking;
 
+
     private ArrayList<Product> productsOnTable;
 
     private Till till ;
@@ -53,7 +54,7 @@
                 type = 'D';
                 closeTable();
                 Invoice.sendInvoice();
-                SendLog();
+                sendLog();
                 till =  new Till().sale(); //invoice amount
 
                 break;
@@ -63,7 +64,7 @@
                 type = 'X';
                 closeTable();
                 Invoice.sendInvoice();
-                SendLog();
+                sendLog();
                 till =  new Till().sale(); //invoice amount
                 till = new Till().cashPaid(); //invoice amount + amount paid in cash
 
@@ -78,10 +79,11 @@
 
         writeToLog.close();
     }
-    public void closeTable() {
+    public void closeTable(int tableNumber) {
+
+    }
 
-
-    }
+        
 
     //closing/deleting booking
     //check if table is vacant
